lib/turnip/builder.rb

```diff
@@ -1,10 +1,15 @@
-require "gherkin/gherkin"
+require "gherkin"
 require 'turnip/node/feature'
 
 module Turnip
   class Builder
     def self.build(feature_file)
-      messages = Gherkin::Gherkin.from_paths([feature_file], include_source: false, include_pickles: false)
+      messages = Gherkin.from_paths(
+        [feature_file],
+        include_source: false,
+        include_gherkin_document: true,
+        include_pickles: true
```

Comment:
Hi @krororo . Thanks to PR !

Is there any reason to change`include_pickles` to `true` ?

Reply:
Sorry... I made a mistake.

Reply:
Fixed.